src/main/resources/META-INF/rewrite/jakarta-ee-9.yml

```diff
@@ -313,6 +331,18 @@ recipeList:
       newGroupId: jakarta.ejb
       newArtifactId: jakarta.ejb-api
       newVersion: latest.release
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: javax.ejb
+      oldArtifactId: javax.ejb
+      newGroupId: jakarta.ejb
+      newArtifactId: jakarta.ejb-api
+      newVersion: latest.release
```

Comment:
I can't seem to find this artifact on https://mvnrepository.com/artifact/javax.ejb/javax.ejb
Are we certain the migration is needed?
```suggestion
```

Reply:
Yeah, this is my mistake, sorry ^^. I confused this for a similar case to javax.inject:javax.inject, but indeed, there is no such artifact.